src/mame/vectorgraphic/vector4.cpp

```diff
@@ -202,12 +193,13 @@ void vector4_state::vector4(machine_config &config)
 	// 7200-0001 page 210 D13, D1
 	clock_device &keyboard_clock(CLOCK(config, "keyboard_clock", _2mclk/26/16));
 	i8251_device &uart0(I8251(config, "uart0", 0));
-	rs232_port_device &rs232keyboard(RS232_PORT(config, "rs232keyboard", default_rs232_devices, "keyboard"));
+	vector4_keyboard_device &v4kbd(VECTOR4_KEYBOARD(config, "rs232keyboard", 0));
 	keyboard_clock.signal_handler().set(uart0, FUNC(i8251_device::write_txc));
 	keyboard_clock.signal_handler().append(uart0, FUNC(i8251_device::write_rxc));
-	uart0.txd_handler().set(rs232keyboard, FUNC(rs232_port_device::write_txd));
-	rs232keyboard.rxd_handler().set(uart0, FUNC(i8251_device::write_rxd));
-	rs232keyboard.set_option_device_input_defaults("keyboard", DEVICE_INPUT_DEFAULTS_NAME(keyboard));
+	uart0.txd_handler().set(v4kbd, FUNC(vector4_keyboard_device::write_rxd));
+	v4kbd.txd_handler().set(uart0, FUNC(i8251_device::write_rxd));
+	// Missing from schematic, but jumper wire present on the board.
+	uart0.write_cts(0);
```

Comment:
You shouldn’t be calling live input members before start time – devices aren’t expected to be able to handle inputs during machine configuration.  Please move this to `machine_start`.